kahuna/public/js/components/gu-date/gu-date.js

```diff
@@ -65,7 +73,7 @@ guDate.directive('guDate', [function () {
             $scope.closeOverlay = () => $scope.showingOverlay = false;
 
             if (angular.isDefined($scope.minDate)) {
-                $scope.dateRounder = (date) => moment(date).endOf('day').toDate();
+                $scope.dateRounder = (date) => moment(date).toDate();
```

Comment:
if we no longer need to round dates, you should be able to get rid of these entirely. all these do now is convert a Date into moment then back into Date